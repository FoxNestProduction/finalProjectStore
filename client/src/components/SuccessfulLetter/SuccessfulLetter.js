import React, { memo } from 'react';
import Box from '@mui/material/Box';
import Typography from '@mui/material/Typography';

const SuccessfulLetter = () => {
  return (
    <Box sx={{ px: 3, pb: 4, display: 'flex', flexDirection: 'column', gap: 3, maxWidth: '560px' }}>
      <Box>
        <img src="./img/sendMail.svg" alt="send mail" />
        {/* <img src="./img/sendMail_1.svg" alt="send mail" /> */}
      </Box>
      <Typography variant="h3" color="primary.main">
        An email with a link has been sent to your email
      </Typography>
      <Typography variant="body3">
        Open this email and click on the link to change your password
      </Typography>
      <Typography sx={{ fontSize: '6px', color: 'text.secondaryGray', fontStyle: 'italic' }}>
        If you did not receive the message, check the spam folder and whether
        the email address is correct and try to send the request again.
      </Typography>
      <Typography variant="body5" color="primary.main">
        The link will only be active for 15 minutes ...
      </Typography>
    </Box>
  );
};

export default memo(SuccessfulLetter);
