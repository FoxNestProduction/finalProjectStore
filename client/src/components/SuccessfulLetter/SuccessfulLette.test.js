import React from 'react';
import { render } from '@testing-library/react';
import SuccessfulLetter from './SuccessfulLetter';

describe('Snapshot test', () => {
  test('should SuccessfulLetter render', () => {
    const { asFragment } = render(<SuccessfulLetter />);
    expect(asFragment()).toMatchSnapshot();
  });
});
