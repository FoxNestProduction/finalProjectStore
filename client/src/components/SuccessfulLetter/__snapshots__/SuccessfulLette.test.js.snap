// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot test should SuccessfulLetter render 1`] = `
<DocumentFragment>
  <div
    class="MuiBox-root css-5nl6tb"
  >
    <div
      class="MuiBox-root css-0"
    >
      <img
        alt="send mail"
        src="./img/sendMail.svg"
      />
    </div>
    <h3
      class="MuiTypography-root MuiTypography-h3 css-1biy274-MuiTypography-root"
    >
      An email with a link has been sent to your email
    </h3>
    <span
      class="MuiTypography-root MuiTypography-body3 css-37jdci-MuiTypography-root"
    >
      Open this email and click on the link to change your password
    </span>
    <p
      class="MuiTypography-root MuiTypography-body1 css-7jadwd-MuiTypography-root"
    >
      If you did not receive the message, check the spam folder and whether the email address is correct and try to send the request again.
    </p>
    <span
      class="MuiTypography-root MuiTypography-body5 css-1kbbegv-MuiTypography-root"
    >
      The link will only be active for 15 minutes ...
    </span>
  </div>
</DocumentFragment>
`;
